scipy/signal/signaltools.py

```diff
@@ -1038,18 +1087,24 @@ def choose_conv_method(in1, in2, mode='full', measure=False):
 
     Notes
     -----
-    For large n, ``measure=False`` is accurate and can quickly determine the
-    fastest method to perform the convolution.  However, this is not as
-    accurate for small n (when any dimension in the input or output is small).
-
-    In practice, we found that this function estimates the faster method up to
-    a multiplicative factor of 5 (i.e., the estimated method is *at most* 5
-    times slower than the fastest method). The estimation values were tuned on
-    an early 2015 MacBook Pro with 8GB RAM but we found that the prediction
-    held *fairly* accurately across different machines.
-
-    If ``measure=True``, time the convolutions. Because this function uses
-    `fftconvolve`, an error will be thrown if it does not support the inputs.
+    Generally, this method is about 90% accurate (99% for 2D signals and 85%
+    accurate for 2D signals for randomly chosen input sizes). For precision
+    (e.g., for different hardware), use ``measure=True`` to find the fastest
+    method by timing the convolution.
+
+    If this funciton is incorrect, the estimated method will very like by less
+    than 5 times slower than the faster method (at least in our
+    experiments). There is a 99.9% chance of this ratio being less than 2 for 2D
+    signals and less than 5 for 1D signals. This function is most inaccurate for
+    fast 1D convolutions taking less that 0.01 seconds. The speed tests were
+    performed on an Amazon EC2 r5a.2xlarge machine.
+
+    This function generalizes fairly decently across hardware. The specific
+    values for this function were tuned on an mid-2014 15-inch MacBook Pro with
+    16GB RAM and a 2.5GHz Intel i7 processor. The speed tests on the EC2
+    machine performed slightly better than the tests on Macbook Pro.
+
+    The convolution is timed the convolutions if ``measure=True``.
     There are cases when `fftconvolve` supports the inputs but this function
     returns `direct` (e.g., to protect against floating point integer
     precision).
```

Comment:
Is this still true after #11044?

Reply:
Yes. Current master: https://github.com/scipy/scipy/blob/c5c6061958e9a0b639acbc41f6eb89ffbf7dd28f/scipy/signal/signaltools.py#L1092-L1100